### PR TITLE
gtk 3.20: Menu and paned fix.

### PR DESCRIPTION
**Pull request by megax**
Done:
- Menu and paned fix.

Sample: 
image

### CODE REVIEW

**Comment by megax** [comment]
@lots0logs ~ Menu and paned fix :)

File: gtk-3.20/scss/widgets/_menu.scss [hunk 1]
```diff
@@ -103,58 +82,111 @@
...
+        check, radio {
+            min-height: 16px;
+            min-width: 16px;
+
+            &:dir(ltr) { margin-right: 7px; }
+            &:dir(rtl) { margin-left: 7px; }
+        }
+
+        separator {
+            padding: 0;
```
Comment (lots0logs) [inline gtk-3.20/scss/widgets/_menu.scss]: @megax The separators are too close to the items they separate. Let's add a few px top/bottom padding if you don't mind. Thanks!

**Comment by megax** [comment]
View list fix. (Border not updated.)
image

**Comment by megax** [comment]
@lots0logs Separator padding not set.

```
        separator {
            padding: 2px;
            border-style: none;
            background-color: shade($menu_bg_color, ($contrast + .1));
        }
```

image

**Comment by lots0logs** [comment]
@megax I found a hack to get some extra space around the separators. I'll push it after merging this :wink:

**Comment by megax** [comment]
Ohhh, ok :)

**Comment by megax** [comment]
Patch done.

**Pull request merged**